about.html

```diff
@@ -0,0 +1,129 @@
+            </li>
+            <li>
+              <a href="#" class="meetup">Buy A Course</a>
+            </li>
+          </ul>
+        </nav>
+        <div id="hamburger" class="menu">
+          <div class="line-1 bar"></div>
+          <div class="contact bar"></div>
+          <div class="line-3 bar"></div>
+        </div>
+      </div>
+    </header>
+    <main>
+      <section class="about-head-line">
+        <h2 class="title-secondary">"Hello! Curious Fellows"</h2>
+        <h1 class="motto">We are bringing Revolution by Evolution to Web</h1>
+        <p class="text-description">
+          we will take your skills to a professional level, take the first
+            step by trying out one of numerous free, self-guided online web
+            development courses  that cover everything from basic programming 
+            introductions to development best practices. Stay Tune and Let Start a wonderful journey. 
+        </p>
+        <p class="contact">
+          Please Contact Us if you have any query.
+        </p>
+        <a href="#" class="link">admin@webevolution.com</a>
+      </section>
+      <section class="container logo-container">
+        <h3 class="title-3"> Web Evolution Idea 2022</h3>
+        <p class="text-3"> The idea of web courese was taken in a seminar about the current state of web and the content available</p>
+          <div class="idea-logo">
+            <img src="images/logo.png" alt="logo" />
+          </div>
+        <div class="our-experience">
+          <h3 class="title-3">Our Success Story</h3>
+          <p class="text-3">
+            We developed many courses in previous year.
+          </p>
+          <div class="experiences">
+            <article class="success">
+              <img
+                src="https://img-b.udemycdn.com/course/240x135/1754098_e0df_3.jpg"
+                alt="Database Desing course"
+                class="experience-img"
+              />
+              <div class="experience-text">
+                <h3 class="exprience-title"> Python Course</h3>
+                <p class="text-3">
+                  We created many courses about python
+                </p>
+              </div>
+            </article>
+            <article class="success">
+              <img
+                src="https://img-b.udemycdn.com/course/240x135/914296_3670_8.jpg"
+                alt="online inventory system"
+                class="experience-img"
+              />
+
+              <div class="experience-text">
+                <h3 class="exprience-title">Web Developement</h3>
+                <p class="text-3">We development many courses about Web Development</p>
+              </div>
+            </article>
+          </div>
+        </div>
+      </section>
+
+    </main>
+    <footer class="footer">
+      <div class="footer-logo">
+        <img src="images/logo.png" alt="Logo " />
+        <p class="company-name">Web Evolution</p>
```

Comment:
- [ ] This line of Html code should not be part of the codebase, kindly remove it as the text that you see in the provided design is part of the logo and not independent text.